modules/tests/src/main/kotlin/com/github/davenury/tests/Changes.kt

```
@@ -44,9 +44,21 @@ class Changes(
 
             val change = createChangeStrategy.createChange(ids, changes)
 
-            val result = changes[ids[0]]!!.introduceChange(change)
+            val result = try {
+                changes[ids[0]]!!.introduceChange(change)
+            } catch (e: Exception) {
```

Should this exception be completely ignored?

---

nope, it should be at least logged :smile: (I'll still blame the brainlag though)